### ISSUE TITLE
Option to view all the headlines

### ISSUE THREAD

**Issue by GADZPMYRJ** [akrennmair/newsbeuter#469]
**Newsbeuter version (copy from `newsbeuter -v`)**:
Compilation date/time: Dec  7 2015 04:47:17
System: Linux 4.8.13-1-ARCH (x86_64)
Compiler: g++ 5.3.0
ncurses: ncurses 6.0.20161224 (compiled with 6.0)
libcurl: libcurl/7.52.1 OpenSSL/1.0.2j zlib/1.2.11 libpsl/0.16.1 (+libicu/58.2) libssh2/1.8.0 (compiled with 7.46.0)
SQLite: 3.16.2 (compiled with 3.9.2)
libxml2: compiled with 2.9.2

**Other info you think is relevant**:
I think an option to view all the headlines from all the feeds would be more than welcome. If all you want is a minimal, easily reproducible setup, creating a query feed/meta feed isn't exactly the cleanest solution. Something like **newsbeuter -x headlines** is so elegant. Just a suggestion.

**Comment by Minoru** [comment]
Wait, you wanted a command for `-x`? I totally misunderstood you, then; sorry! Query feeds aren't a solution for this particular problem.

Can you describe your use-case for that? Newsbeuter has notifications, and there's also [rsstail](https://github.com/flok99/rsstail).

**Comment by GADZPMYRJ** [comment]
No, I just want to open newsbeuter and have all my headlines displayed. I suggested a possibility. If it's not the right way, then feel free to choose another way.  Maybe something like **startup-headlines yes** in config?

**Comment by Minoru** [comment]
Ah, so I understood you right. Okay.

Okay, so suppose there is such setting, and you set it to "yes". You start Newsbeuter, read a few articles and decide you want to see the feed X. How would you do that? (With #402 and query feeds you'd press q while in the article list, which will bring you back to feed list, where you'll select feed X and go on reading just it.)

**Comment by GADZPMYRJ** [comment]
A suggestion: **builtin** meta feed named _'headlines'_ and the option to open any feed we want automatically at startup. This **builtin** meta feed will have its own **builtin id** documented in manual pages.

**Comment by Minoru** [comment]
Okay, I'll consider that when I work on #402, and I'll close this issue when I make the decision. Thanks!

**Comment by Minoru** [comment]
Link #297.
